go/private/actions/stdlib.bzl

```diff
@@ -59,6 +59,12 @@ def _has_precompiled_stdlib(version_string):
     dot = minor.find(".")
     if dot != -1:
         minor = minor[:dot]
+    rc = minor.find("rc")
+    if rc != -1:
+        minor = minor[:rc]
+    beta = minor.find("beta")
+    if beta != -1:
+        minor = minor[:beta]
```

Comment:
> ~Looks good to me with one comment, we should be able to merge this right away.~
> 
> ~@linzhp Could you test at Uber once the comment has been resolved?~
> 
> Missed the coverage test failure, will also look into it.

I think the coverage test is the same issue on the boringcrypto cl: that we need to make the path to "Tool" relative on 1.19+.

Reply:
I ran it locally with extra output and it complained about `-test.coverprofile` not being supported when built without coverage. Since the CI is passing for the other PR, I rather suspect something changed with respect to coverage instrumentation in Go 1.20.

Reply:
Ah, got it, that will require a closer look...

Reply:
Ok so that error was due to the flag-flip setting the GOEXPERIMENT coverageredesign by default. We'll need to eventually update the way coverage works in rules_go, but for now we can set GOEXPERIMENT=nocoverageredesign. I'll try to find a way to set that to get this change moving forward. It might be easiest to plumb something through the same way we do in the boringcrypto cl